CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1540,4 +1540,20 @@ local backpackType, healthType = Enum.CoreGuiType.Backpack, Enum.CoreGuiType.Hea
 OnCoreGuiChanged(backpackType, StarterGui:GetCoreGuiEnabled(backpackType))
 OnCoreGuiChanged(healthType, StarterGui:GetCoreGuiEnabled(healthType))
 
+local RobloxGui = game:GetService("CoreGui"):WaitForChild("RobloxGui")
+local UserInputService = game:GetService("UserInputService")
+local UISChanged
+local function OnVREnabled(prop)
+	if prop == "VREnabled" and UserInputService.VREnabled then
+		OnCoreGuiChanged(backpackType, false)
+		OnCoreGuiChanged(healthType, false)
+		spawn(function() require(RobloxGui.Modules.BackpackScript3D) end)
+		UISChanged:disconnect()
+
+		BackpackScript.TopbarEnabledChanged = function() end
```

Comment:
This function seems like a hack, lets sit down and figure out a solid solution to disabling the backpack